### ISSUE TITLE
new technique: Hierarchical indexing

### ISSUE THREAD

**Issue by szafarnia** [openMetadataInitiative/openMINDS_instances#96]
Dear all,

Could you please add "Hierarchical indexing" as a new technique to KG?

Many thanks in advance.

Best,
Sara

**Comment by lzehl** [comment]
@UlrikeS91 correct, "cluster validation analysis" should be under schema type AnalysisTechnique

@tgbugs would you agree registering this analysis technique (which will be a super class, since there are multiple actual analysis technique of actually doing this validation)
